dlrover/python/common/global_context.py

```diff
@@ -143,6 +144,7 @@ def __init__(self):
         self.npu_per_node = DefaultValues.NPU_NUM_PER_NODE
         # pre-check args
         self.pre_check_operators = DefaultValues.PRE_CHECK_OPS
+        self.max_relaunch_count = DefaultValues.MAX_RELAUNCH_COUNT
```

Comment:
u need to modify this part: https://github.com/intelligent-machine-learning/dlrover/blob/30ae58ca998d2bc0e598254a5a2c3df6bc7574ed/dlrover/python/master/node/dist_job_manager.py#L166